posthog/models/organization.py

```diff
@@ -125,6 +121,10 @@ class PluginsAccessLevel(models.IntegerChoices):
     for_internal_metrics: models.BooleanField = models.BooleanField(default=False)
     is_member_join_email_enabled: models.BooleanField = models.BooleanField(default=True)
     enforce_2fa: models.BooleanField = models.BooleanField(null=True, blank=True)
+    # Scoring mechanism defined here: https://github.com/PostHog/product-internal/pull/564
+    trusted_customer_scores: models.JSONField = models.JSONField(default=dict, null=True, blank=True)
+    trusted_customer_score_overrides: models.JSONField = models.JSONField(default=dict, null=True, blank=True)
```

Comment:
What is the purpose of this field?

Reply:
This seemed like the easiest way to support manual score overrides (e.g. through django admin) without interfering with the automatic score calculation mechanism. We store two dictionaries and merge them at the point of use in quota_limiting.

Reply:
I think a single map is probably better and achieves the same result - it's only incremented if they meet the criteria for a higher score, and is never programmatically decremented. Someone (eg CS) can go in and set it manually if they want. If they manually downgrade someone, then it won't automatically upgrade again if we don't have the `highest_paid_bill` in here and just go based on the current bill.

Reply:
So the situation is - a customer is paying a bunch of money so would have a higher score, but we don't like them or they commonly make implementation errors so we manually downgrade them to drop when they hit their quota. 

IDK how often that will happen. This is only relevant for people who are on the free plan or have billing limits set, and are prone to hitting those limits.

Reply:
Yes, that's true, but if we maintain an automatic score recalculation system (and I think we should do this in billing/customer any time a subscription changes or a new invoice is completd), their score would simply be re-upped to what it was. 

For example, if they're on level 7 but get manually overwritten to 0, then upgrade to the teams plan, the automatic scoring mechanism would mark them as 10.